src/browser/BrowserShared.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright (C) 2020 KeePassXC Team <team@keepassxc.org>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "BrowserShared.h"
+#include "config-keepassx.h"
+
+#include <QProcessEnvironment>
+#include <QStandardPaths>
+
+namespace BrowserShared
+{
+    QString localServerPath()
+    {
+        const QString serverName = QStringLiteral("/org.keepassxc.KeePassXC.BrowserServer");
```

Comment:
If we merge my config refactor branch first, you could get the prefix dynamically from a global application property.

Reply:
Perfect

Reply:
What's the status of this?

Reply:
I am actually not sure what you were going after with this comment. Can you rephrase or implement yourself? I pushed my final changes that addresses everything else.

Reply:
You wanted to get this name dynamically from the app property that is set globally in main.cpp.